tmt/steps/execute/internal.py

```diff
@@ -168,7 +168,7 @@ def execute(self, test: Test, guest: Guest,
         guest.push(
             source=test_wrapper_filepath,
             destination=test_wrapper_filepath,
-            options=["-p", "--chmod=755"])
+            options=["-s", "-p", "--chmod=755"])
```

Comment:
Would it make sense to add `-s` to `DEFAULT_RSYNC_PUSH_OPTIONS` and `DEFAULT_RSYNC_PULL_OPTIONS`? It seems like a feature tmt might need to have enabled in general.

Reply:
I've put it to every rsync command